bb-pr

```diff
@@ -319,8 +319,9 @@ action_checkout() {
 
 git_switch_default() {
   if [[ -n "$GIT_REMOTE_DEFAULT" && "$GIT_LOCAL_WORKING_BRANCH" ]]; then
-    git checkout "$GIT_REMOTE_DEFAULT"
+    git switch "$GIT_REMOTE_DEFAULT"
     git pull
+    git remote prune origin
```

Comment:
Maybe prune isn't the correct answer:

```
URL: git@bitbucket.org:....git
 * [pruned] origin/feat/...
 * [pruned] origin/feat/....
 * [pruned] origin/feat/...
 * [pruned] origin/feat/...
 * [pruned] origin/feature/...
Deleted branch feat/... (was xxxx).
````

Reply:
Might be best to leave it as an exercise for the user to do a `git fetch origin --prune --prune-tags`

It'll be that 1% edge case where you really burn yourself.

Reply:
@quotidian-ennui What about:

```shell
git branch --remotes --delete $(git rev-parse --abbrev-ref --symbolic-full-name @{u})
```